### ISSUE TITLE
If the message contains new line characters, the message won't be shown

### ISSUE THREAD

**Issue by KaiKikuchi** [blay09/BetterMinecraftChat#26]
Mod Version: 1.10.2-4.1.6
Modpack: Resonant Rise 4 Dev a.1.1.0

If the server uses the new line character (\n), the entire message won't be shown to all clients with BetterMinecraftChat.

This is a sample Sponge code snippet that helps reproducing the issue:

``` java
@Listener(order=Order.POST)
public void onPlayerJoin(ClientConnectionEvent.Join event) {
  event.getTargetEntity().sendMessage(Text.of("This is the first line", Text.NEW_LINE, "This is the second line"));
}
```

**Comment by BlayTheNinth** [comment]
Mod is being rewritten to work better at the moment, this will likely be fixed automatically when I'm done

**Comment by KaiKikuchi** [comment]
Thanks. Keep up the good work!

**Comment by BlayTheNinth** [comment]
This was fixed as part of the rewrite for Minecraft 1.11.2.
